crates/holochain_p2p/src/hc_report.rs

```diff
@@ -0,0 +1,599 @@
+use holochain_types::report::{ReportEntry, ReportEntryFetchedOps};
+use kitsune2_api::*;
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex, Weak};
+
+const MOD: &str = "HCREPORT";
```

Comment:
This shows up in the "advanced" config, and we seem to have used camel case for other module names, could this be too for consistency?

Reply:
Done: https://github.com/holochain/holochain/pull/5274/commits/b2e3a7f1e458910c8ad983267e17e54660d1509c